pkg/crc/config/validations.go

```diff
@@ -22,14 +22,6 @@ func ValidateBool(value interface{}) (bool, string) {
 	return false, "must be true or false"
 }
 
-// ValidateDriver checks if driver is valid in the config
-func ValidateDriver(value interface{}) (bool, string) {
-	if err := validation.ValidateDriver(value.(string)); err != nil {
```

Comment:
after this change, `validation.ValidateDriver` is unused, so this method can also be removed.